template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/ConfigUtils.java

```diff
@@ -51,4 +52,15 @@ public static ApiClusterTemplateVariable variable(String name, String value) {
                 .findFirst();
     }
 
-}
+    public static Optional<StorageLocationView> getStorageLocationForServiceProperty(TemplatePreparationObject source, String serviceProperty) {
+        return source.getFileSystemConfigurationView()
+                .get().getLocations().stream()
+                .filter(s -> s.getProperty().equalsIgnoreCase(serviceProperty))
+                .findFirst();
+    }
+
+    public static String getSafetyValveProperty(String key, String value) {
+        String safetyValveFormat = "<property><name>%s</name><value>%s</value></property>";
```

Comment:
I suggest extracting this as a private constant.

Reply:
Fixed